PersonStageThree/src/com/beyond/person/core/utils/DanceType.java

```diff
@@ -0,0 +1,9 @@
+package com.beyond.person.core.utils;
```

Comment:
@maria-salmasian please add enumeration package in utils

Reply:
added